pkgs/applications/version-management/git-and-tools/bump2version/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, isPy27, pytest, testfixtures, lib }:
+
+buildPythonPackage rec {
```

use buildPythonApplication as it will isolate the command from polluting a user's environment with the python dependencies

```suggestion
buildPythonApplication rec {
```